src/DIRAC/WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1904,10 +1915,10 @@ def setHeartBeatData(self, jobID, dynamicDataDict):
             return ret
         e_jobID = ret["Value"]
 
-        req = "UPDATE Jobs SET HeartBeatTime=UTC_TIMESTAMP(), Status='%s' WHERE JobID=%s" % (JobStatus.RUNNING, e_jobID)
+        req = f"UPDATE Jobs SET HeartBeatTime=UTC_TIMESTAMP(), Status='{JobStatus.RUNNING}' WHERE JobID={e_jobID}"
         result = self._update(req)
         if not result["OK"]:
-            return S_ERROR("Failed to set the heart beat time: " + result["Message"])
+            return S_ERROR(f"Failed to set the heart beat time: " + result["Message"])
```

Comment:
As you made this an f-string I presume you meant:
```suggestion
            return S_ERROR(f"Failed to set the heart beat time: {result['Message']}")
```